solutions/2.1-bar_and_fill_between.py

```diff
@@ -23,8 +24,10 @@
 
 # Now you're on your own!
 
+#making sublots
 fig, ax = plt.subplots()
 
+#Styling the Plot a little ...
```

Comment:
It doesn't just style; it does the actual plotting too. So this comment needs to be expanded a bit more.